tests/php/Forms/RequiredFieldsTest.php

```diff
@@ -322,4 +324,71 @@ public function testHasOneRelationFieldInterfaceValidation(string $className)
         // '1' passes required field validation
         $this->assertTrue($validator->php(['TestField' => '1']));
     }
+
+    public static function provideAllowWhitespaceOnly(): array
+    {
+        return [
+            'no-ws' => [
+                'value' => 'abc',
+                'expectedTrue' => true,
+                'expectedFalse' => true,
+            ],
+            'left-ws' => [
+                'value' => 'abc',
+                'expectedTrue' => true,
+                'expectedFalse' => true,
+            ],
+            'right-ws' => [
+                'value' => 'abc',
+                'expectedTrue' => true,
+                'expectedFalse' => true,
+            ],
+            'both-ws' => [
+                'value' => 'abc',
+                'expectedTrue' => true,
+                'expectedFalse' => true,
+            ],
```

Comment:
Seems like the values for these should be different? They're all just `'abc'`

Reply:
opps, have updated